packages/util/src/BridgeApi.ts

```diff
@@ -394,4 +397,43 @@ export class BridgeApi extends BaseApi {
     const data = (await (this.api.rpc as any).ethBridge.getApprovals(hashes, this.externalNetwork)).toJSON();
     return this.getData(data);
   }
+
+  /**
+   * Creates a subscription to bridge request status
+   * @param networkId external network id
+   * @param hash sora or evm transaction hash
+   * @returns BridgeRequest status
+   */
+  public subscribeOnRequestStatus(networkId: number, hash: string): Observable<BridgeTxStatus | null> {
```

Comment:
may it be null?

Reply:
in case ETH-SORA transfer: while sora transaction is waiting 30 blocks confirmations, the related data get by ethereum hash is null